config/jobs/kubernetes/sig-testing/verify.yaml

```diff
@@ -17,7 +17,10 @@ presubmits:
       preset-dind-enabled: "true"
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:v20210226-c001921-master
+      # TODO(releng): Use the 'master' variant once kubernetes/kubernetes has
+      #               been updated to go1.16
+      #               ref: https://github.com/kubernetes/kubernetes/pull/98572
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20210302-c18942a-go-canary
```

Comment:
we should not be using the canary tag for the real job?

we should be updating the real image, after testing in the canary with the latest version in a canary/experimental job. we auto push -latest images.

these changes left other jobs not updated.

Reply:
if we want to have a special image to try go early for an extended period that should get it's own job

we should have a manually invoked job that uses the standard image track at the -latest tag with always pull instead of the date-sha version (which lets us test the latest version of the standard image prior to an image bump here)

the primary job should keep using the date-sha version so we can roll forward/back without waiting for an image push, and track the versions in git.